tileview/src/main/java/com/qozix/tileview/widgets/ZoomPanLayout.java

```diff
@@ -713,7 +716,9 @@ public boolean onDown( MotionEvent event ) {
 
   @Override
   public boolean onFling( MotionEvent event1, MotionEvent event2, float velocityX, float velocityY ) {
-    mScroller.fling( getScrollX(), getScrollY(), (int) -velocityX, (int) -velocityY, 0, getScrollLimitX(), 0, getScrollLimitY() );
+    getScroller().fling( getScrollX(), getScrollY(), (int) -velocityX, (int) -velocityY,
```

Comment:
@krabo0om this is why I only added the top and left: I had forgotten them before.